docs/changelog.rst

```diff
@@ -13,6 +13,8 @@ Added
 - New raster method for adding gdal_compliant() attributes to xarray object.
 - Function ``to_datetimeindex`` in available preprocess functions for xr.open_dataset in the data adapter.
 - Function ``remove_duplicates`` in available preprocess functions for xr.open_dataset in the data adapter.
+- New ``DataCatalog.from_predefined_catalogs`` and ``DataCatalog.from_archive`` to support predefined data catalogs and archive 
+  in a generic way through the data/predefined_catalogs.yml file.
```

Comment:
Maybe good to also add it the list of newly available predefined catalogs / versions?

Reply:
We'll keep a separate changelog file for the data catalogs as those are not connected to the hydromt versions, see #156